examples/lib/pi_monte_carlo.dart

```diff
@@ -0,0 +1,54 @@
+// ignore_for_file: type_annotate_public_apis
+// Note: this DartPad version has a few extra lines of code,
+// marked "web-only", to make the sample execution nicer on the web
+// (approximations of π are added to the DOM in addition to being
+// printed on the console).
+
+import 'dart:async'; //!tip("import")
+import 'dart:html'; //!web-only
+import 'dart:math' show Random; //!tip("dart:math")
+//!web-only
+int numIterations = 500; //!web-only
+
+main() async { //!tip("main()")
+  print('Compute π using the Monte Carlo method.'); //!tip("π")
+  var output = querySelector("#value-of-pi"); //!web-only
+  await for (var estimate in computePi().take(numIterations)) { //!tip("await")
+    print('π ≅ $estimate'); //!tip("$estimate")
+    output.text = estimate.toStringAsFixed(5); //!web-only
+    await window.animationFrame; //!web-only
+  }
+}
+
+/// Generates a stream of increasingly accurate estimates of π. //!tip("///")
+Stream<double> computePi({int batch: 100000}) async* { //!tip("async*") //!tip("{int batch: 100000}") //!tip("<double>")
```

Comment:
Was the change from batch of 1M to 100K intentional? No problem either way, just checking.

Reply:
This was to match the original sources in the published DartPad: https://dartpad.dartlang.org/fb763a4a770b5cdd896982e10ccf4118